README.md

```diff
@@ -37,22 +36,26 @@ using GoogleMapsApi.StaticMaps;
 using GoogleMapsApi.StaticMaps.Entities;
 
 //Static class use (Directions) (Can be made from static/instance class)
-DirectionsRequest directionsRequest = new DirectionsRequest()
+var directionsRequest = new DirectionsRequest()
 {
     Origin = "NYC, 5th and 39",
     Destination = "Philladephia, Chesnut and Wallnut",
+    IsSSL = true,
+    ApiKey = googleAPIKey,
 };
 
-DirectionsResponse directions = GoogleMaps.Directions.Query(directionsRequest);
+var directions = await GoogleMaps.Directions.QueryAsync(directionsRequest);
 Console.WriteLine(directions);
 
 //Instance class use (Geocode)  (Can be made from static/instance class)
-GeocodingRequest geocodeRequest = new GeocodingRequest()
+var geocodeRequest = new GeocodingRequest()
 {
-    Address = "new york city",
+    Address = "NYC, 5th and 39",
+    IsSSL = true,
+    ApiKey = googleAPIKey,
 };
 var geocodingEngine = GoogleMaps.Geocode;
-GeocodingResponse geocode = geocodingEngine.Query(geocodeRequest);
```

Comment:
Is this should be part of this commit? Seems like unrelated.

Reply:
`IEngineFacade` only have `QueryAsync` method, so `GeocodingResponse geocode = geocodingEngine.Query(geocodeRequest);`  should be changed to  `var geocode = await geocodingEngine.QueryAsync(geocodeRequest);`